packages/components/pie-button/test/component/pie-button.spec.ts

```diff
@@ -41,7 +41,7 @@ test('should emit an event when clicked', async ({ mount }) => {
         default: 'Click me!'
       },
       on: {
-        CustomEvent: (data: string) => messages.push(data),
+        onCustomEvent: (data: string) => messages.push(data),
```

Comment:
Maybe there's something I'm missing but again, the name change causes this to read in a bit of a confusing way in my opinion. `on: onCustomEvent` seems a little strange (I could be completely missing something here)

Reply:
fixed by reverting this change 👍